### PR TITLE
CP-9316: Buttons and Glasses

### PR DESCRIPTION
**Pull request by onghwan**
## Description

**Ticket: [CP-9316]** 

* Implement [k2-alpine glasses](https://www.figma.com/design/opZ4C1UGzcoGRjxE4ZIE3J/K2-Alpine?node-id=1-721&node-type=frame&t=6VRzxt6bXnFjH9T7-11).
* Implement [k2-alpine buttons](https://www.figma.com/design/opZ4C1UGzcoGRjxE4ZIE3J/K2-Alpine?node-id=7-1465&t=6VRzxt6bXnFjH9T7-4).
* Add storybooks for glasses and buttons.
* Fix scrolls in the new app when AppSwitcher is used.

### Glasses
https://github.com/user-attachments/assets/81f55388-c1cc-471e-adad-c537feb04969

### Buttons

https://github.com/user-attachments/assets/7c33f32e-0748-424b-a25c-ac0841311017

### Buttons and Glasses in the new app
https://github.com/user-attachments/assets/ecdb020b-8a95-4d10-b825-c55e57799fff

## Checklist
- [X] I have performed a self-review of my code
- [X] I have verified the code works
- [ ] I have added/updated necessary unit tests 
- [ ] I have updated the documentation

[CP-9316]: https://ava-labs.atlassian.net/browse/CP-9316?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/k2-alpine/src/components/Button/Button.tsx [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+): string | undefined => {
+  if (disabled) {
+    return theme.isDark
+      ? overlayColor(alpha(colors.$neutralWhite, 0.3), colors.$neutral850)
+      : overlayColor(alpha(colors.$neutral850, 0.3), colors.$neutralWhite)
+  }
+
+  switch (type) {
+    case 'primary':
+      return theme.isDark ? colors.$neutralWhite : colors.$neutral850
```
Comment (atn4z7) [inline packages/k2-alpine/src/components/Button/Button.tsx:204]: ideally we shouldn't need to do this and should just use surfacePrimary. if it is like this in the design then we should ping Todd/Michael

File: packages/k2-alpine/src/components/Button/Button.tsx [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+  StyleProp,
+  StyleSheet,
+  TouchableOpacity,
+  ViewStyle
+} from 'react-native'
+import { useDripsyTheme as useTheme } from 'dripsy'
+import { Text, View } from '../Primitives'
+import { Icons } from '../../theme/tokens/Icons'
+import {
+  colors,
```
Comment (atn4z7) [inline packages/k2-alpine/src/components/Button/Button.tsx:13]: is it possible to not use colors directly?

File: packages/k2-alpine/src/components/GlassView/GlassView.tsx [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+        return alpha('#FFFFFF', 0.6)
+      case 'light2':
+        return alpha('#A1A1AA', 0.25)
+      case 'dark':
+        return alpha('#28282E', 0.85)
+      case 'dark2':
+        return alpha('#818189', 0.6)
+      case 'dark3':
+        return alpha('#C5C5C8', 0.25)
+    }
```
Comment (atn4z7) [inline packages/k2-alpine/src/components/GlassView/GlassView.tsx:28]: do these stay constant for both dark/light theme?

File: packages/core-mobile/AppSwitcher.tsx [hunk 4]
```diff
@@ -27,7 +29,14 @@ export const AppSwitcher = (): React.JSX.Element => {
     () =>
       PanResponder.create({
         onStartShouldSetPanResponder: (evt, gestureState) => {
-          if (gestureState.numberActiveTouches === 2) {
+          return (
```
Comment (atn4z7) [inline packages/core-mobile/AppSwitcher.tsx:32]: nice fix!

File: packages/k2-alpine/src/components/Button/Button.tsx (hunk 2 above)
Comment (onghwan) [reply packages/k2-alpine/src/components/Button/Button.tsx:13]: good point. fixed.

File: packages/k2-alpine/src/components/GlassView/GlassView.tsx (hunk 3 above)
Comment (onghwan) [reply packages/k2-alpine/src/components/GlassView/GlassView.tsx:28]: yes. light/light2 type should be only used in light mode, and dark/dark2/dark3 are for dark mode.

File: packages/k2-alpine/src/components/Button/Button.tsx [hunk 5]
```diff
@@ -0,0 +1,237 @@
...
+          alpha(darkModeColors.$surfaceSecondary, 0.3),
+          lightModeColors.$surfacePrimary
+        )
+  }
+
+  switch (type) {
+    case 'primary':
+      return theme.isDark
+        ? lightModeColors.$surfacePrimary
+        : darkModeColors.$surfacePrimary
```
Comment (atn4z7) [inline packages/k2-alpine/src/components/Button/Button.tsx:208]: we shouldn't reference the colors directly anywhere if possible. can we use theme.colors?

**Review by atn4z7 — APPROVED** [review]
nice work!!!!

**Pull request merged**